src/Test/PdbUtilities/Metadata/ILVisualizer.cs

```diff
@@ -406,10 +406,19 @@ public void VisualizeHeader(StringBuilder sb, int codeSize, int maxStack, Immuta
                             break;
 
                         case OperandType.InlineString:
-                            sb.Append(' ');
-                            sb.Append(VisualizeUserString(ReadUInt32(ilBytes, ref curIndex)));
-                            break;
+                            {
+                                sb.Append(' ');
+                                uint pseudoToken = ReadUInt32(ilBytes, ref curIndex);
+                                // Check for an encoding of that indices the current module's module version ID.
+                                if (pseudoToken == 0x80000000)
```

Comment:
Using the symbolic constant would be better, but the metadata writer didn't seem to be available here. If it's important, I'll figure out how to make the constant available.